figures/mapping-histology-labels.Rmd

```diff
@@ -55,10 +55,10 @@ library(RColorBrewer)
 
 We should perform some minimal checking to make sure the provided release is _at least_ at version 22.
 ```{r}
-minimum_allowed_release <- 22
+minimum_allowed_release <- 23
```

Comment:
I feel both ways about this change...
- The code will run on v22, so that could be interpreted as minimal version
- We'd _prefer_ for the code to run on v23, which is another interpretation.

I leave this call to you! But, just make sure the text on line 56 matches how the code ends up.

Reply:
Updated the comment and also took the following note out:

> **Note: This notebook was originally written with `release-v21-20210820`. Future releases may necessitate revisiting some of this code.**

Seemed a little unnecessary (future releases may necessitate revisiting _any code_ in the repo) and potentially confusing.